activestorage/test/models/variant_test.rb

```diff
@@ -158,9 +158,10 @@ class ActiveStorage::VariantTest < ActiveSupport::TestCase
   end
 
   test "variation of invariable blob" do
-    assert_raises ActiveStorage::InvariableError do
+    e = assert_raises ActiveStorage::InvariableError do
```

Comment:
```suggestion
    assert_raises ActiveStorage::InvariableError, match: /Can't transform blob with ID=\d+ and content_type=application\/pdf/ do
```

You can use `assert_raises` with a match:
https://edgeapi.rubyonrails.org/classes/ActiveSupport/Testing/Assertions.html#method-i-assert_raises

Reply:
Thanks I didn't know this.
This made the lines really long though.
Shorting the match didn't make much sense either.

Reply:
OK, I've shortened the match.